test/mlir/test/gc/Dialect/Microkernel/merge-branch-microkernel-context.mlir

```diff
@@ -0,0 +1,278 @@
+// RUN: gc-opt %s -early-dispatch-microkernel -convert-microkernel-to-dnnl-func -cse -merge-branch-microkernel-context -split-input-file | FileCheck %s
```

Comment:
We'd better to test as less pass as possible in a unit test. Is it possible to check merge-branch optimization only?

Reply:
Fixed, now it run `-merge-branch-microkernel-context` only.